src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/BlogPostsController.java

```diff
@@ -0,0 +1,30 @@
+package pl.tomaszdziurko.jvm_bloggers.blog_posts;
+
+import java.io.PrintWriter;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.sun.syndication.io.SyndFeedOutput;
+
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+
+@RestController
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class BlogPostsController {
+
+    private final AggregatedRssFeedProducer rssProducer;
+
+    @SneakyThrows
```

Comment:
Hmm... I think for now it's good enough. When we introduce some kind of monitoring or metrics, we could think about adding @ControllerAdvice to get all exceptions in one place.